### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kenodegard**
Include more details regarding the purpose of this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+  * 💭 Share ideas about new features and changes you'd like to see
+  * 🗣️ Engage with other community members
 
 Please start by creating a discussion first before you open an issue directly.
 
 Welcome others and be open-minded. Remember that this is a community we build together 💪.
+
+## GitHub Workflows
+
+We have defined a number of GitHub Workflows that are synced out to other repos within the Conda GitHub Org. These workflows accomplishes:
```
Comment (beeankha) [inline README.md:26]: Change last bit to: 
```
These workflows accomplish:
```

Comment (kenodegard) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -1,18 +1,30 @@
-## 👋 Welcome!
+# Conda GitHub Org Infrastructure
+
+Centralized location for infrastructure codes as well as any infrastructure related inquiries or discussions.
```
Comment (beeankha) [inline README.md:3]: Maybe change the beginning of this sentence to "This is a centralized location for..."

File: README.md [hunk 3]
```diff
@@ -1,18 +1,30 @@
...
+
+<table><tr><td><b>⚠️ This is not a forum for end user questions.</td></tr></table>
```
Comment (beeankha) [inline README.md:5]: end-user (include the dash)

File: README.md [hunk 4]
```diff
@@ -1,18 +1,30 @@
...
-  * etc.
+## 👋 Welcome!
 
-**NOTE: This is not a forum for end user questions!**
+We are using [GitHub discussions in this repo](https://github.com/conda/infra/discussions) as a place for [Conda core members and maintainers](https://github.com/conda-incubator/governance#teams--roles) to raise issues about the Conda community infrastructure such as:
+  * 📦 Package hosting (e.g. conda-forge channel on anaconda.org)
+  * 👤  Conda GitHub Org admin questions and issues
+  * 🔄 shared GitHub Workflows synced to other repos in Conda GitHub Org
+  * 🧪 Test suite and development setup issues
+  * ❔ Anything that does not fall into any other project specific buckets (e.g. the [conda issues](https://github.com/conda/conda/issues) or [conda-build issues](https://github.com/conda/conda-build/issues))
```
Comment (beeankha) [inline README.md:14]: project-specific (include a dash)

File: README.md [hunk 5]
```diff
@@ -1,18 +1,30 @@
...
-  * Test suite and development setup issues
-  * Anything that does not fall into any other project specific buckets (e.g. the [conda issues](https://github.com/conda/conda/issues) or [conda-build issues](https://github.com/conda/conda-build/issues))
-  * etc.
+## 👋 Welcome!
 
-**NOTE: This is not a forum for end user questions!**
+We are using [GitHub discussions in this repo](https://github.com/conda/infra/discussions) as a place for [Conda core members and maintainers](https://github.com/conda-incubator/governance#teams--roles) to raise issues about the Conda community infrastructure such as:
+  * 📦 Package hosting (e.g. conda-forge channel on anaconda.org)
+  * 👤  Conda GitHub Org admin questions and issues
+  * 🔄 shared GitHub Workflows synced to other repos in Conda GitHub Org
```
Comment (beeankha) [inline README.md:12]: Nitpick: Capitalize "shared" for consistency

**Review by beeankha — APPROVED** [review]
Left some minor comments, overall changes lgtm! :shipit:

File: README.md (hunk 2 above)
Comment (kenodegard) [reply README.md]: ✔️

File: README.md [hunk 6]
```diff
@@ -1,18 +1,30 @@
...
-We’re using [GitHub discussions in this repo](https://github.com/conda/infra/discussions) as a place to provide a place for [Conda core members and maintainers](https://github.com/conda-incubator/governance#teams--roles) to raise issues about the Conda community infrastructure such as:
-  * Package hosting (e.g. conda-forge channel on anaconda.org)
-  * GitHub organization admin questions and issues
-  * Test suite and development setup issues
-  * Anything that does not fall into any other project specific buckets (e.g. the [conda issues](https://github.com/conda/conda/issues) or [conda-build issues](https://github.com/conda/conda-build/issues))
-  * etc.
+## 👋 Welcome!
 
-**NOTE: This is not a forum for end user questions!**
+We are using [GitHub discussions in this repo](https://github.com/conda/infra/discussions) as a place for [Conda core members and maintainers](https://github.com/conda-incubator/governance#teams--roles) to raise issues about the Conda community infrastructure such as:
```
Comment (jezdez) [inline README.md:9]: That reminds me, that this repo isn't only for Conda core members but was originally created for all community members that want to raise issues around hosting etc, e.g. Bioconda and conda-forge maintainers. Let's fix this while we're here.

**Review by jezdez — APPROVED** [review]
Just a tiny nit about something that was already in the text and can be updated while we're there.

File: README.md (hunk 6 above)
Comment (kenodegard) [reply README.md:9]: Which phrasing?

1. "...Conda core members and community maintainers..."
2. "...Conda core members and community members..."
3. "...community maintainers..."
4. "...community members..."

I'm partial for either of the "community maintainers" options, I think "community members" is too broad.

Comment (jezdez) [reply README.md:9]: Let's go with "Conda core and community members" that should be broad enough, right?

**Pull request merged**